web/admin/pages/users/[did].vue

```diff
@@ -5,6 +5,10 @@ import { ProfileViewDetailed } from "@atproto/api/dist/client/types/app/bsky/act
 
 const api = await useAPI();
 
+const error = ref<{
+  rawMessage: string;
+}>(null);
```

Comment:
**suggestion**: we can just use a `string` here as type, which is simpler and gets us the same result. Then, an empty string would be no error.